tests/tests/test_conf.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.test import TestCase
+
+from pipeline.conf import PipelineSettings
+
+class TestSettings(TestCase):
+
+    def test_3unicode(self):
+        s = PipelineSettings(dict(), DEFAULTS={ "PIPELINE_FOO_BINARY": "env actualprogram" })
```

Comment:
Ultimately ``PipelineSettings`` is just a wrapper around django settings, you should even be able to drop your ``DEFAULTS`` change: https://github.com/cyberdelia/django-pipeline/blob/master/tests/tests/test_compressor.py#L96-L97